spec/anycable/pubsub_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe Anycable::PubSub do
+  let(:config) { Anycable.config }
+  subject { described_class.new }
+
+  context "with sentinel" do
+    it do
+      expect(Redis).to receive(:new).with(url: config.redis_url, sentinels: config.redis_sentinels)
+      subject
+    end
+  end
+
+  context "without sentinel" do
+    before do
```

Comment:
You should add `after { Anycable.config.reload }` here to make sure that every test works with _non-dirty_ config.

Reply:
With `after { Anycable.config.reload }` I have strange errors like:

```
subscriptions #unsubscribe invokes #unsubscribed for channel
     Failure/Error: subject { service.command(request) }

     GRPC::Unavailable:
       14:Connect Failed
     # ./vendor/bundle/gems/grpc-1.6.0-universal-darwin/src/ruby/lib/grpc/generic/active_call.rb:31:in `check_status'
     # ./vendor/bundle/gems/grpc-1.6.0-universal-darwin/src/ruby/lib/grpc/generic/active_call.rb:171:in `attach_status_results_and_complete_call'
     # ./vendor/bundle/gems/grpc-1.6.0-universal-darwin/src/ruby/lib/grpc/generic/active_call.rb:363:in `request_response'
     # ./vendor/bundle/gems/grpc-1.6.0-universal-darwin/src/ruby/lib/grpc/generic/client_stub.rb:152:in `request_response'
     # ./vendor/bundle/gems/grpc-1.6.0-universal-darwin/src/ruby/lib/grpc/generic/service.rb:170:in `block (3 levels) in rpc_stub_class'
     # ./spec/integrations/subscriptions_spec.rb:74:in `block (3 levels) in <top (required)>'
     # ./spec/integrations/subscriptions_spec.rb:83:in `block (4 levels) in <top (required)>'
     # ./spec/integrations/subscriptions_spec.rb:83:in `block (3 levels) in <top (required)>'
```

You can check it on Travis

Reply:
There was a problem with config reloading (we set some configuration in our `spec_helper.rb` and it's not available after reload). I've fixed this in the upcoming version branch (`v0.5.0-dev`). Just rebase against it and everything should work.

And, please, update the changelog.

Reply:
Done!